src/pages/index.js

```diff
@@ -113,10 +114,7 @@ const renderCard = (data) => {
     "#card-template"
   );
 
-  const renderCard = (data) => {
-    const cardElement = createCard(data);
-    section.addItem(cardElement);
-  };
+  document.querySelector(".cards__list").prepend(cardElement.getView());
```

Comment:
`document.querySelector(".cards__list").prepend`



`section.addItem(cardElement);` should be used here


Only class `Section` should add any card to the DOM.